examples/internal-load-balancer/variables.tf

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# These variables are expected to be passed in by the operator
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "project" {
+  description = "The project ID to create the resources in."
+  default     = "dev-sandbox-228703"
+}
+
+variable "region" {
+  description = "The region to create the resources in."
+  default     = "europe-north1"
+}
+
+variable "zone" {
+  description = "The availability zone to create the sample compute instances in. Must within the region specified in 'var.region'"
+  default     = "europe-north1-a"
+}
```

Comment:
Reminder to remove the defaults before merge.

Reply:
Good point!